tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature

```diff
@@ -2886,4 +2886,32 @@ Feature: Create a share link for a resource
       | createOnly       | view                 |
       | createOnly       | edit                 |
       | createOnly       | upload               |
-      | blocksDownload   | blocksDownload       |
\ No newline at end of file
+      | blocksDownload   | blocksDownload       |
+
+
+  Scenario: remove link share of a file in project drive using permissions endpoint
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "to share" to "textfile.txt"
+    And user "Alice" has created the following resource link share:
+      | resource        | textfile.txt |
+      | space           | projectSpace |
+      | permissionsRole | view         |
+      | password        | %public%     |
+    When user "Alice" delets the link share of file "textfile.txt" inside space "projectSpace" using the permissions endpoint of the Graph API
+    Then the HTTP status code should be "204"
+
+  
+  Scenario: remove link share of a file in project drive using permissions endpoint
```

Comment:
user removes access to a file in project space in link share
user removes access to a folder in project space in link share